### PR TITLE
Add libqglviewer

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
I am not  sure what the license specifier for this package should look like.

<details>
  <summary>License - click to expand</summary>
libQGLViewer GPL Exception
==========================

Additional rights granted beyond the GPL (the "Exception").

As a special exception to the terms and conditions of GPL version 2.0 or 
GPL version 3.0, Gilles Debunne hereby grants you the rights described below,
provided you agree to the terms and conditions in this Exception, including 
its obligations and restrictions on use.

Nothing in this Exception gives you or anyone else the right to change the
licensing terms of the libQGLViewer Open Source Edition.

Below, "Licensed Software" shall refer to the software licensed under the GPL
version 2.0 or GPL version 3.0 and this exception.

1) The right to use Open Source Licenses not compatible with the GNU
General Public License version 2.0 or GNU General Public License version
3.0: You may link software (hereafter referred to as "Your Software")
against the Licensed Software and/or distribute binaries of Your Software
linked against the Licensed Software, provided that:

A) Your Software is licensed under one of the following licenses:


License name                            Version(s)/Copyright Date
Academic Free License                   2.0, 2.1, 3.0
Apache Software License                 1.0 or 1.1
Apache License                          2.0
Apple Public Source License             2.0
BSD license                             "July 22 1999"
Common Development and Distribution
  License (CDDL)                        1.0
Common Public License                   1.0
Eclipse Public License                  1.0
GNU Library or "Lesser"
General Public License (LGPL)           2.0, 2.1, 3.0
Jabber Open Source License              1.0
Mozilla Public License (MPL)            1.0 or 1.1
Open Software License                   2.0, 3.0
PHP License                             3.0
Python Software Foundation License      2.1.1
Q Public License                        1.0
Sleepycat License                       "1999"
W3C License                             "2001"
X11 License                             X11R6.6
Zope Public License                     2.0, 2.1


and

B) You must, on request, make a complete package including the complete 
    source code of Your Software (as defined in the GNU General Public 
    License version 2, section 3, but excluding anything excluded by the
    special exception in the same section) available to Gilles Debunne
    under the same license as that granted to other recipients of the 
    source code of Your Software.

and

C) Your or any other contributor's rights to:

        i) distribute the source code of Your Software to anyone for
        any purpose;

        and

        ii) publicly discuss the development project for Your
        Software and its goals in any form and in any forum

are not prohibited by any legal instrument, including but not limited to
contracts, non-disclosure agreements, and employee contracts.


2) The right to link non-Open Source applications with versions of the 
Licensed Software: You may link applications with versions of the Licensed 
Software, provided that such applications have been developed and are deployed
in accordance with the terms and conditions of the libQGLViewer Commercial 
License Agreement.
</details>

**Comment by Tobias-Fischer** [comment]
Hi @conda-forge/help-c-cpp this is ready for review. It would be great if you could let me know what to do about the license (see above). Many thanks!

**Comment by Tobias-Fischer** [comment]
Kind ping @conda-forge/help-c-cpp :)

/cc @wolfv - this is for octovis in robostack

**Comment by xhochy** [comment]
Recipe looks fine from the C++/packaging side but I'm no SPDX expert so would appreciate a second review there.

File: recipes/libqglviewer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+test:
+  commands:
+    - test -f $PREFIX/include/QGLViewer/qglviewer.h  # [unix]
+    - test -f $PREFIX/lib/libQGLViewer$SHLIB_EXT     # [unix]
+    - if exist %PREFIX%\\Library\\include\\QGLViewer\qglviewer.h (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://libqglviewer.com
+  license: GPL-2.0-or-later
```
Comment (isuruf) [inline recipes/libqglviewer/meta.yaml:52]:
```suggestion
  license: GPL-2.0-only OR GPL-3.0-only
```

**Pull request merged**